File: Unidad 5/JAVA/P7_U5.java
import java.util.Scanner;
public class P7_U5
{
	public static void main(String[] args) {
	  Scanner sc = new Scanner(System.in);
	    
        System.out.println("Ingrese el subtotal de la factura:");
        double sub = sc.nextDouble();

        double iva = calcular_IVA(sub);
        double total = calcular_Total(sub, iva);

        System.out.println("Subtotal: $" + sub);
        System.out.println("IVA: $" + iva);
        System.out.println("Total: $" + total);
    }
    public static double calcular_IVA(double sub) {
        double porcentaje_IVA = 0.16;
        return sub * porcentaje_IVA;
    }

    public static double calcular_Total(double sub, double iva) {
        return sub + iva;
    }
}

File: Unidad 5/C#/P7_U5.cs
using System;

class P7_U5
{
    static void Main()
    {
        Console.WriteLine("Ingrese el subtotal de la factura:");
        double sub = double.Parse(Console.ReadLine());

        double iva = Calcular_IVA(sub);
        double total = Calcular_Total(sub, iva);

        Console.WriteLine($"Subtotal: ${sub}");
        Console.WriteLine($"IVA: ${iva}");
        Console.WriteLine($"Total: ${total}");
    }

    static double Calcular_IVA(double sub)
    {
        double porcentaje_IVA = 0.16;
        return sub * porcentaje_IVA;
    }

    static double Calcular_Total(double sub, double iva)
    {
        return sub + iva;
    }
}